plugins/osdn/common.go

```diff
@@ -51,6 +51,8 @@ type OsdnController struct {
 // Called by plug factory functions to initialize the generic plugin instance
 func (oc *OsdnController) BaseInit(registry *Registry, pluginHooks PluginHooks, multitenant bool, hostname string, selfIP string) error {
 
+	log.Infof("Starting with configured hostname '%s' (IP '%s')", hostname, selfIP)
+
```

Comment:
Yeah, but that's after the fixups.  I'd like to see what the configured/command-line values are too.